packages/muze-legend/src/legend/step-legend.js

```diff
@@ -51,7 +51,7 @@ export default class StepLegend extends SimpleLegend {
      *
      * @memberof StepLegend
      */
-    dataFromScale (scale) {
+    dataFromScale (scale, context) {
```

Comment:
If this function needs contextual information, shouldn't it be a part of the class?